### PR TITLE
Add updater.png to resources and set it as window icon. Working 'Cont…

### PR DESCRIPTION
**Pull request by FreeSlave**
…inue reading' urls in news text and pointing hand cursor on Read More button

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -16,6 +16,7 @@ int main(int argc, char *argv[])
     QCoreApplication::setApplicationName("Unvanquished Updater");
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
     QApplication app(argc, argv);
+	app.setWindowIcon(QIcon(":resources/updater.png"));
```
Comment (DolceTriade) [inline main.cpp]: nit: indent is wrong.

**Review by DolceTriade — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (illwieckz) [reply main.cpp]: tabs forever

**Comment by FreeSlave** [comment]
Forcibly re-pushed with right indent

**Review by DolceTriade — APPROVED** [review]

**Comment by DolceTriade** [comment]
Thanks!

**Pull request merged**